docs: troubleshooting backlog items (#9474)
langflow-ai/langflow: docs/docs/Integrations/Nvidia/integrations-nvidia-ingest.mdx

| extract_infographics | Extract Infographics | Extract infographics from document. Default: false. |
| text_depth | Text Depth | The level at which text is extracted. Options: 'document', 'page', 'block', 'line', 'span'. Default: `page`. |
| split_text | Split Text | Split text into smaller chunks. Default: true. |

> | chunk_size | Chunk Size | The number of tokens per chunk. Default: `500`. **Note:** When using NVIDIA embedding models like `nvidia/nv-embed-v1`, you may need to use smaller chunk sizes (`500` or less) to avoid tokenization errors. For more information, see [Tokenization errors due to chunk size](/components-processing#chunk-size). |

Reviewer: ```suggestion
| chunk_size | Chunk Size | The number of tokens per chunk. Default: `500`. Make sure the chunk size is compatible with your embedding model. For more information, see [Tokenization errors due to chunk size](/components-processing#chunk-size). |
```
